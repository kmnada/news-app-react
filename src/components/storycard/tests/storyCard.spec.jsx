import React from "react";
import { shallow } from "enzyme";
import StoryCard from "../storyCard";

test("should render StoryCard component", () => {
  const wrapper = shallow(
      <StoryCard />
  );
  expect(wrapper).toMatchSnapshot();
});
