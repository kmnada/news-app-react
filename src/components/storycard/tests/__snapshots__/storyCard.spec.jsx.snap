// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render StoryCard component 1`] = `ShallowWrapper {}`;
